docs/components/all.md

```diff
@@ -14,14 +14,16 @@ type: component
 Using this package, all JSX email components can be accessed from a single import. For folks who prefer that style, versus importing individual component packages.
 
 ```jsx
-import { Button, Heading } from '@jsx-email/all';
+import { Button, Heading } from "@jsx-email/all";
```

Comment:
if you run `moon repo:lint` then this will revert back to single ticks. looks like your editor might have changed that